.pre-commit-config.yaml

```diff
@@ -96,7 +96,7 @@ repos:
             attrs,
             types-requests,
             types-python-dateutil,
-            apache-airflow,
+            apache-airflow~=2.9,
```

Comment:
Can we add a TODO to revert this once we validate the provider with Airflow 3?

Reply:
Sure, @pankajkoti , thanks a lot. Addressed in d980049bc74fdcdc4f394c775c558e88f704c45b

Reply:
I also created a follow up issue to track Airflow 3 support that references this: https://github.com/astronomer/astro-provider-anyscale/issues/87